mupen64plus-video-rice/upstream/src/Video.cpp

```diff
@@ -51,6 +51,7 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 static void (*l_DebugCallback)(void *, int, const char *) = NULL;
 static void *l_DebugCallContext = NULL;
```

Comment:
For this, I will probably have to fork this repo to this project:

https://github.com/mupen64plus/mupen64plus-video-rice